File: C/26. 删除有序数组中的重复项.c
int removeDuplicates(int* nums, int numsSize){
  int l=0,r=1;
  while(r<numsSize){
      if(nums[l]!=nums[r]){
          nums[l+1]=nums[r];
          l++;
          r++;
      }else {
          r++;
      }
  }
  return l+1;
}

File: doublePoint/26. 删除有序数组中的重复项.js
/**
 * @param {number[]} nums
 * @return {number}
 */
 var removeDuplicates = function(nums) {
  let l=0,r=1;
  while(r<nums.length){
      if(nums[l]!==nums[r]){
          nums[l+1]=nums[r];
          l++;
          r++;
      }else {
       r++;
      }
  } 
  return l+1;
};